modules/nrf_wifi/bus/qspi_if.c

```diff
@@ -1000,28 +993,23 @@ static int qspi_nor_configure(const struct device *dev)
 static int qspi_nor_init(const struct device *dev)
 {
 #ifdef CONFIG_PINCTRL
-	int ret = pinctrl_apply_state(QSPI_IF_BUS_PINCTRL_DT_DEV_CONFIG_GET,
-				      PINCTRL_STATE_DEFAULT);
+	int ret = pinctrl_apply_state(QSPI_IF_BUS_PINCTRL_DT_DEV_CONFIG_GET, PINCTRL_STATE_DEFAULT);
```

Comment:
Did you run this through clang-format? It does these unnecessary changes, please remove them.